Source/FolioReaderConfig.swift

```diff
@@ -11,6 +11,7 @@ import UIKit
 public enum FolioReaderScrollDirection: Int {
     case vertical
     case horizontal
+	case sectionHorizontalContentVertical
```

Comment:
This is a really interesting addition 👍 

Some points that I have noticed:

- [ ] The internal pages counter that show minutes and pages left is not working when using this option;
- [ ] When using this option I think we have to disable `canChangeScrollDirection`, so by default this option has not ability to change the layout;
- [ ] When open close book, the current reading position is not restored.